content/contact.md

```diff
@@ -12,7 +12,7 @@ draft: false
 
 Patrick Hogan - [patrick.hogan@nasa.gov](mailto://patrick.hogan@nasa.gov)
```

Comment:
The link should be "mailto:patrick.hogan@nasa.gov". The forward slashes are erroneous.